atm.js

```diff
@@ -1,3 +1,110 @@
-$(document).ready(function(){
```

Comment:
Ideally we'd want to keep our jQuery code inside of a `$(document).ready()` block for a couple of reasons. First we may not always have control of the html document to re-order script tags, (that could be a different team's baby) and our code would fail since the document wasn't fully loaded. But this also creates a scope for our variables and keeps them from polluting the global namespace, or worse colliding with other globals that might have been defined by another part of the app.